packages/inventory/src/Stocktake/DetailView/ContentArea/ContentArea.tsx

```diff
@@ -19,7 +20,15 @@ const Expando = ({
   const expandoColumns = useExpansionColumns();
 
   if ('lines' in rowData && rowData.lines.length > 1) {
-    return <MiniTable rows={rowData.lines} columns={expandoColumns} />;
+    return (
+      <MiniTable
+        rows={rowData.lines}
+        columns={expandoColumns}
+        queryParamsStore={createQueryParamsStore<StocktakeLineFragment>({
+          initialSortBy: { key: 'expiryDate' },
```

Comment:
not that this will do anything, we're not refetching here. really just a nicety, and potentially useful if the hook is called and the sort options requested for a manual sort.